src/planscape/forsys/merge_polygons_test.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+from forsys.merge_polygons import merge_polygons
+from django.test import TestCase
+from django.contrib.gis.geos import Polygon
+from planscape import settings
+
+
+class MergePolygonsTest(TestCase):
+    def setUp(self):
+        self.xorig = -2116971
+        self.yorig = 2100954
+
+        self.side = 300
+        self.xscale = self.side
+        self.yscale = -self.side
+
+        self.srid = settings.CRS_FOR_RASTERS
+
+        self.margin = np.sqrt(2) * self.side / 2
+
+    def test_merges_contiguous_polygons(self) -> None:
+        p1 = self._create_polygon(((0, 0), (0, 5), (5, 5), (5, 0), (0, 0)))
+        # p2 is contiguous with p1.
+        p2 = self._create_polygon(((5, 0), (5, 3), (8, 3), (8, 0), (5, 0)))
+        poly = merge_polygons([p1, p2], self.margin)
```

Comment:
it seems like margin doesn't have any effect when the polygons are contiguous. based on the image in the PR description, i thought it might try to simplify those corners

Reply:
not quite!
I hope my function description helps clarify this functionality.
I should also update tests to make this more understandable.

Reply:
Updated tests.